### PR TITLE
fix(ci): add tag 'apisix-runtime'

### PR DESCRIPTION
**Pull request by zll600**
try to fix: https://github.com/api7/apisix-build-tools/actions/runs/7985520662/job/21804131057

### CODE REVIEW

**Review by membphis — DISMISSED** [review]
seems good to me

**Comment by membphis** [comment]
it seems we have to merge this to master branch, and release a new tag?

**Comment by zll600** [comment]
@membphis `apisix-runtime` is removed by https://github.com/api7/apisix-build-tools/pull/358/files#diff-ed6e73a8ff832e1eb0070922f731ac6e66bf7858ed0df0e8f9c5d0983a21a270L44, maybe we need help from @AlinsRan

**Comment by AlinsRan** [comment]
> @membphis `apisix-runtime` is removed by https://github.com/api7/apisix-build-tools/pull/358/files#diff-ed6e73a8ff832e1eb0070922f731ac6e66bf7858ed0df0e8f9c5d0983a21a270L44, maybe we need help from @AlinsRan

This is a packaged and published CI. As the apisix runtime is only an intermediate product when publishing, we can release it if necessary,

Or we can filter this tag.

File: .github/workflows/publish-deb.yml [hunk 1]
```diff
@@ -2,8 +2,8 @@ name: Build and Publish DEB Package
 
 on:
   push:
-    tags:
-      - "**"
+    tags-ignore:
+      - "apisix-runtime/*"
```
Comment (AlinsRan) [inline .github/workflows/publish-deb.yml:6]: How about this?
```suggestion
    tags:
      - "apisix/*"
```

Comment (zll600) [reply .github/workflows/publish-deb.yml:6]: Don't we need `apisix-base` and `apisix-dashboard`? https://github.com/api7/apisix-build-tools/blob/389e59601d25d232daf180ee25ab83272902fdfb/.github/workflows/publish.yml#L62

Comment (AlinsRan) [reply .github/workflows/publish-deb.yml:6]: No need for apisix-base.
The dashboard has not been used for a long time.

I didn't find any tags with `dashboard/*`.

Comment (zll600) [reply .github/workflows/publish-deb.yml:6]: OK

File: .github/workflows/publish-deb.yml [hunk 2]
```diff
@@ -2,8 +2,8 @@ name: Build and Publish DEB Package
 
 on:
   push:
-    tags:
-      - "**"
+    tags-ignore:
```
Comment (AlinsRan) [inline .github/workflows/publish-deb.yml:5]: ditto

**Review by AlinsRan — APPROVED** [review]

**Review by monkeyDluffy6017 — APPROVED** [review]

**Pull request merged**